yabt/target_utils.py

```diff
@@ -241,13 +263,21 @@ def compute_hash(self, build_context):
         m = md5()
         m.update(self.json(build_context).encode('utf8'))
         self._hash = m.hexdigest()
+        m.update(self.test_json(build_context).encode('utf8'))
```

Comment:
is it intentional that the test hash is additive on top of the "regular" hash? if so, please explain in a comment

Reply:
Not intentional. Hope it's OK now.